cdap-ui/app/features/overview/templates/data-apps.html

```diff
@@ -0,0 +1,95 @@
+<div class="row">
+  <div class="col-xs-6">
+    <div ng-if="datasets.length > 0 || streams.length > 0">
+      <h2 class="text-uppercase"> Data </h2>
+      <div class="row">
+        <div class="col-xs-6">
+          <h3> Deployed datasets: </h3>
+        </div>
+        <div class="col-xs-6">
+          <div class="pull-right">
+            <a ui-sref="data.list" class="btn btn-default">
+              <span class="fa fa-bars"></span>
+              All Data
+            </a>
+          </div>
+        </div>
+      </div>
+
+      <ul class="list-group" >
+        <li class="list-group-item" ng-repeat="stream in streams track by $index">
+          <a ui-sref="streams.detail.overview({streamId: stream.name})">
+            <strong ng-bind="stream.name | caskCapitalizeFilter">  </strong>
+            <div> <p ng-bind="'Type: ' + (stream.type)"></p> </div>
+          </a>
+        </li>
+        <li class="list-group-item" ng-repeat="dataset in datasets track by $index">
```

Comment:
If we can move the list-group-item class to the <a> tag then the entire list item becomes clickable, instead of just the link tag (which in this case the text)